rule-types/github/dockerfile_no_latest_tag.yaml

```
@@ -29,7 +29,6 @@ def:
   ingest:
     type: git
     git:
```

I'm fairly sure leaving this empty errors when creating the ruletype. It should be an empty object instead
```suggestion
    git: {}
```

---

It should accept nil, at least that's how I've been creating rule types lately.

---

You're right